### DISCUSSION TITLE
Necessary to update AUTOMATIC 1111?

### DISCUSSION THREAD

**Discussion by 9wow** [lllyasviel/stable-diffusion-webui-forge#648] [Q&A] [answered]
My Forge is using the AUTOMATIC 1111 venv. So, should I keep A 1111 updated or does it not matter; just using Forge update is fine?

**Comment by ostap667inbox** [comment, marked as answer]
A1111 and Forge update separately, they just share a common VENV folder. In case of problems with A1111 or Forge, you can even delete the VENV folder and when you start A1111 or Forge it will re-generate.
So the answer is no, you don't need to keep updating A1111 to be able to update Forge.
